cli.py

```diff
@@ -393,8 +409,11 @@ def attack(name):
 
     args = parser.parse_args()
 
-    # Unless a database has been defined for ingest, default to <profile>.db
-    if "profile" in args and args.database is None:
+    # If environment variables are in use and the database is undefined, default to general.db
+    if args.env and args.database is None:
+        args.database = "general.db"
+    # If environment variable are not being used and the database is undefined default to <profile>.db
+    elif "profile" in args and args.database is None:
```

Comment:
Hi Jake,

#12 is a fantastic suggestion, thank you so much for this! 

If you wouldn't mind, could you make the following change and I will happily accept the PR:


```suggestion
    # Unless a database has been defined for ingest, default to <profile>.db
    if 'database' in args and args.database is None:
        args.database = f"{args.profile}.db"
```
This will default to `default.db` in the absence of a profile and allow `--profile` to be used in conjunction with `--env` to specify a database,

Reply:
Ah, I like that solution much more